components/landing/TheQuickStart/Algorithms.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <section class="algorithms">
+    <div class="algorithms__container">
+      <h3>Get the SDK</h3>
+      <div class="algorithms__section">
+        <CodeSnippet
+          :code-lines="[installCode]"
+          :code-snippet-title="'TODO: update this'"
+          :code-snippet-location="'quick-start-algorithms'"
+        />
+      </div>
+      <h3>Test some algorithms</h3>
+      <bx-tabs container :value="activeTabValue">
+        <bx-tab
+          v-for="(algorithm, index) in codeExamples"
+          :key="index"
+          :target="index"
+          :value="index"
+        >
+          {{ algorithm.name }}
+        </bx-tab>
+      </bx-tabs>
+      <div
+        v-for="(algorithm, index) in codeExamples"
+        :id="index"
+        :key="index"
+        class="algorithms__section"
+        role="tabpanel"
+        :aria-labelledby="index"
+      >
+        <CodeSnippet
+          :code-lines="algorithm.fullCode"
+          :code-snippet-title="algorithm.name"
+          :code-snippet-location="'quick-start-algorithms'"
+        />
+      </div>
+    </div>
+  </section>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
+import CodeSnippet from '~/components/ui/CodeSnippet.vue'
+import '@carbon/web-components/es/components/tabs/index.js'
+
+type codeExample = {
+  name: string
+  runMethod: string
+  fullCode: string[]
+}
+
+@Component({
+  components: { CodeSnippet }
+})
+export default class Algorithms extends Vue {
+  @Prop(String) installCode!: string
+  @Prop(Array) codeExamples!: codeExample[]
+
+  activeTabValue = 0
+
+  updated () {
+    // use the first available algorithm
```

Comment:
note: this isn't working as I intended, will revisit.

Reply:
It's probably a good idea to add a TODO comment.